src/Facade/Facade.csproj

```diff
@@ -25,4 +25,10 @@
     <ProjectReference Include="..\Resources\Resources.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.Routing">
+      <HintPath>..\..\..\..\Program Files\dotnet\packs\Microsoft.AspNetCore.App.Ref\6.0.4\ref\net6.0\Microsoft.AspNetCore.Routing.dll</HintPath>
```

Comment:
what does this do?

Reply:
resharper must have snuck that it at some point. will remove